tools/tsconfig.tools.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this is only one special case for nx, as it compiles down TS source of workspace generators - it uses this file in particular

Reply:
I'm still a bit confused by intention of both `tsconfig.tools.json` and `tsconfig.lib.json`, why are both necessary ?

Reply:
sorry about that. 

- `tsconfig.tools.json` is used to run `tsc` when executing `nx workspace-generator <generator-name>` runner 
- `tsconfig.lib.json` is standard part of every package ( 100% converged to nx ), which is used for production builds (transpile/rollup) and as source of truth for type checking implementation files. 
  - I added it to `/tools` to keep the default boilerplate consistent